### PR TITLE
Experiment with OpenNLP Language detection

### PR DESCRIPTION
**Pull request by gerashegalov**

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -9,8 +9,7 @@ dependencies {
     compile "com.googlecode.libphonenumber:carrier:$googleCarrierVersion"
 
     // Optimaize language detection
-    compile "com.salesforce.transmogrifai:language-detector:$optimaizeLangDetectorVersion"
-
+    compile "com.optimaize.languagedetector:language-detector:$optimaizeLangDetectorVersion"
```
Comment (tovbinm) [inline core/build.gradle:12]: There was some fixes I did into Optimaize language detector. I think it was related to Guava.

**Comment by tovbinm** [comment]
Let us know if it works better or worse than Optimaize one.

**Comment by tovbinm** [comment]
It seems that it has more languages supported. Is this correct?

File: core/build.gradle (hunk 1 above)
Comment (tovbinm) [reply core/build.gradle:12]: https://github.com/optimaize/language-detector/pull/74

Comment (gerashegalov) [reply core/build.gradle:12]: I was hoping to get rid of this fork by shading if necessary.

Comment (tovbinm) [reply core/build.gradle:12]: Sure. No development is happening on that repo, so not a big deal imo.

**Comment by Jauntbox** [comment]
Cool - curious to see how this compares. Another one we could try is FastText, which also has a language detection module: https://github.com/facebookresearch/fastText/#full-documentation

**Pull request closed without merging**